<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferenceDialog</class>
 <widget class="QDialog" name="PreferenceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>520</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Preference</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabStyle">
      <attribute name="title">
       <string>Style</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Code Editor Font:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QFontComboBox" name="cbCodeEditorFont">
           <property name="fontFilters">
            <set>QFontComboBox::MonospacedFonts</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Font Pixel Size:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="sbFontPointSize">
           <property name="minimum">
            <number>6</number>
           </property>
           <property name="maximum">
            <number>48</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Markdown Engine:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="cbMarkdownEngine">
           <item>
            <property name="text">
             <string>Goldmark</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Lute</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Preview Theme:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="cbPreviewTheme"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Preview Mode:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="cbPreviewMode">
           <item>
            <property name="text">
             <string>Wechat Public Account Article</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Blog Post</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Code Block Style:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="cbCodeBlockStyle">
           <item>
            <property name="text">
             <string>abap</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>algol</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>algol_nu</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>api</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>arduino</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>autumn</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>borland</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>bw</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>colorful</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>dracula</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>emacs</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>friendly</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>fruity</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>github</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>igor</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>lovelace</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>manni</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>monokai</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>monokailight</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>murphy</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>native</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>paraiso-dark</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>paraiso-light</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>pastie</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>perldoc</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>pygments</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>rainbow_dash</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>rrt</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>solarized-dark</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>solarized-dark256</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>solarized-light</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>swapoff</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>tango</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>trac</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>vim</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>vs</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>xcode</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="previewThemeEditorContainer" native="true"/>
       </item>
       <item>
        <widget class="QCheckBox" name="cbEnableLineNumbers">
         <property name="text">
          <string>Enable Line Numbers in Code Block</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbMacTerminalStyleCodeBlock">
         <property name="text">
          <string>Mac Terminal Style Code Block</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabTranslation">
      <attribute name="title">
       <string>Translation</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Translation Service</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="cbGoogleTranslate">
            <property name="text">
             <string>Google Translate</string>
            </property>
            <property name="icon">
             <iconset resource="karenmeu.qrc">
              <normaloff>:/rc/images/google.png</normaloff>:/rc/images/google.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbBaiduTranslate">
            <property name="text">
             <string>Baidu Translate</string>
            </property>
            <property name="icon">
             <iconset resource="karenmeu.qrc">
              <normaloff>:/rc/images/baidu.png</normaloff>:/rc/images/baidu.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbYoudaoTranslate">
            <property name="text">
             <string>Youdao Translate</string>
            </property>
            <property name="icon">
             <iconset resource="karenmeu.qrc">
              <normaloff>:/rc/images/youdao.png</normaloff>:/rc/images/youdao.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbSogouTranslate">
            <property name="text">
             <string>Sogou Translate</string>
            </property>
            <property name="icon">
             <iconset resource="karenmeu.qrc">
              <normaloff>:/rc/images/sogou.png</normaloff>:/rc/images/sogou.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Translate Timeout(s):</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="sbTranslateTimeout">
         <property name="maximum">
          <number>90</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabCloud">
      <attribute name="title">
       <string>Cloud</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Server:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="edtCloudServerAddress"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Username:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="edtCloudUsername"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Password:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="edtCloudPassword">
         <property name="echoMode">
          <enum>QLineEdit::PasswordEchoOnEdit</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="karenmeu.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferenceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferenceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
